x-pack/plugins/session_view/server/routes/process_events_route.ts

```diff
@@ -87,6 +87,7 @@ export const fetchEventsAndScopedAlerts = async (
                   { term: { [EVENT_ACTION]: 'fork' } },
                   { term: { [EVENT_ACTION]: 'exec' } },
                   { term: { [EVENT_ACTION]: 'end' } },
+                  { term: { [EVENT_ACTION]: 'text_output' } },
```

Comment:
While the simplicity of this is nice, and we might be able to get away with this approach for most scenarios, it's possible a single process could have up to (max_kilobytes_per_process_event: default 512kb) per process (and a user could increase this at will). This could end up causing the page of process events to be quite large (only to show an indicator of output). Ideally we would do an aggregatation on process.entity_id (where event.action: 'text_output'), and then use that result set to maybe fake some lightweight (no text data) "process" events to be sent back with the page of events. Like we do for alerts.